wiki/Brand_identity_guidelines/pt-br.md

```diff
@@ -0,0 +1,72 @@
+# Diretrizes de identidade da marca
+
+As diretrizes de identidade da marca são um conjunto de padrões para branding de projetos relacionados ao osu!. Todo uso não requer aprovação.
+
+[Baixe o kit de design aqui.](https://drive.google.com/file/d/1TmUot5nu49p71icz4u3G68njLAQOeQrG/view?usp=sharing) O kit de design linkado contém alguns recursos desatualizados. Note também que o PDF incluído está desatualizado e esta página deve ser utilizada em seu lugar.
+
+## Logo osu! cookie
+
+### Restrições de uso do cookie
+
+#### Cor única
+
+![](img/usage-single-colour.png)
+
+Essa é a versão de cor única do logo osu! cookie. Esta versão do logo é muito versátil e pode ser adaptada para vários estilos de design.
+
+Por favor, use o arquivo original como base e não crie o logo do zero.
+
+#### Colorida
+
+![](img/usage-full-colour.png)
+
+Essa é a versão colorida do logo osu! cookie. A sombra sutil faz parte do logo colorida.
+
+Por favor, use o logo como fornecido pelo kit de design sem nenhuma modificação.
+
+### Espaço livre
+
+![](img/clear-space-area.png)
+
+Por favor, dê ao cookie algum espaço para respirar. Use o "o" em "osu!" como uma medida de espaço.
+
+### Restrições de uso do cookie de cor única
+
+Uma vez que osu! é mantido pela comunidade, o logo osu! cookie é projetado para ser simples e versátil; ele pode se adaptar para vários designs com facilidade. Não há restrição de cores para o cookie.
+
+![](img/restrictions-good-single.png)
+
+- O cookie deve sempre ser agudo.
+- Você pode usar qualquer cor para o cookie.
+- Você pode adicionar um gradiente linear ao cookie. Gradientes radiais não são recomendados já que o anel e o elemento do meio poderia aparecer em cores completamente diferentes.
```

Comment:
```suggestion
- Você pode adicionar um gradiente linear ao cookie. Gradientes radiais não são recomendados já que o anel e o elemento do meio podem aparecer em cores completamente diferentes.
```
Or at least `poderiam` in case you do not agree, because plural.